Modules/posixmodule.c

```diff
@@ -9255,59 +9306,64 @@ os_preadv_impl(PyObject *module, int fd, PyObject *buffers, Py_off_t offset,
                int flags)
 /*[clinic end generated code: output=26fc9c6e58e7ada5 input=4173919dc1f7ed99]*/
 {
-    Py_ssize_t cnt, n;
-    int async_err = 0;
-    struct iovec *iov;
-    Py_buffer *buf;
+    if (HAVE_PREADV_RUNTIME) {
```

Comment:
I'd just ignore the compiler warning here, as this function will be removed from the module when HAVE_PREADV_RUNTIME is false.

Reply:
ok